### ISSUE TITLE
No npm package?

### ISSUE THREAD

**Issue by ghost** [hkirat/notification-logger#2]

**Comment by mikeerickson** [comment]
@sleewoo If the author @hkirat does not want to create npm package, I will fork the project and create it.  

In addition, I can extend the project to also supply an icon property (see related issue [#1](https://github.com/hkirat/notification-logger/issues/1))

**Comment by hkirat** [comment]
I have creating the npm package. Updating it soon @mikeerickson

**Comment by mikeerickson** [comment]
@hkirat cool, thanks!

**Comment by mikeerickson** [comment]
@hkirat will you be adding the `icon` property as well?

**Comment by hkirat** [comment]
I'll be modifying the website and Readme for now. I'll take that up later. You can file for a P.R. if you wish.

**Comment by hkirat** [comment]
https://github.com/hkirat/notification-logger/commit/20c022b92b420569330734990d1438b22ba8ef66

**Comment by mikeerickson** [comment]
@hkirat I will issue PR after you have completed the npm package.

**Comment by hkirat** [comment]
Done with the npm package @mikeerickson 
https://www.npmjs.com/package/notification-logger

**Comment by mikeerickson** [comment]
@hkirat OK, sounds good.  I will add the icon property later today and get a PR submitted by EOD. Thank you for your efforts.

**Comment by mikeerickson** [comment]
@sleewoo @hkirat Just to be clear and properly set expectations.  

The creation of the npm package is only for the requirement to install package via npm as opposed to downloading the archive.  The method of use remains the same as if you were to download.

TL;DR the npm package install does NOT provide a commonJS format (or UMD) which would allow it to be installed into your application using ES5 `require` or ES6 `import`
